beacon_node/eth1/src/block_cache.rs

```diff
@@ -36,6 +37,25 @@ impl Eth1Block {
     }
 }
 
+#[derive(Encode, Decode, Clone)]
+pub struct SszBlockCache {
```

Comment:
Could we just implement `Encode, Decode` on `BlockCache`?

I've only been using this `Ssz..` structs when we can't just impl on the base type (e.g., it has a `HashMap`)

Reply:
Yup makes sense